tests/NCronJob.Tests/NCronJob.Tests.csproj

```diff
@@ -6,6 +6,7 @@
         <Nullable>enable</Nullable>
         <IsPackable>false</IsPackable>
```

Comment:
Does it need `IsPackable` and `IsTestProject` anymore. I guessed I removed this in my other projects.

Reply:
@linkdotnet Seems to still be of mainstream usage:
- https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-platform-integration-dotnet-test#dotnet-test---microsofttestingplatform-mode
- https://github.com/xunit/xunit/blob/main/src/Directory.Build.props#L103

Reply:
Let’s keep it. Maybe I was a donkey for removing this in my other projects